manifests/machineconfigcontroller/deployment.yaml

```diff
@@ -27,10 +27,42 @@ spec:
             cpu: 20m
             memory: 50Mi
         terminationMessagePolicy: FallbackToLogsOnError
+      - name: oauth-proxy
```

Comment:
Because I didn't know about it :) 

It sounds like we probably should use it -- I set this up to mirror what the machine-config-daemon does, since we already had precedent, and it uses oauth-proxy. I probably won't change the machine-config-controller to operate this way today since I don't fully understand all of the ramifications, but I have added a JIRA ticket to investigate this as part of our monitoring improvement epic https://issues.redhat.com/browse/MCO-128